cookbook/extend-entities.rst

```diff
@@ -71,15 +71,20 @@ Sulu `User` class.
 .. warning::
 
     Your Entity can have own properties, but they should have at least default values.
-    Otherwise the normal features of Sulu could crash (like the 
+    Otherwise the normal features of Sulu could crash (like the
     `sulu:security:user:create` command).
 
+    The `@ORM\\Table` annotation need to match the exist table else you will get strange
+    errors when query data from it.
```

Comment:
```suggestion
.. warning::

    The `@ORM\\Table` annotation on your entity must match the table of the extended entity. 
    Otherwise, doctrine might run into errors when querying data of the entity.
```